packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -447,3 +448,33 @@ const webSocketApi = new apigwv2.WebSocketApi(this, 'mywsapi',{
 });
 ```
 
+## OpenAPI Definition
+
+CDK supports creating an HTTP API by importing an OpenAPI definition file. It currently supports OpenAPI v2.0 and OpenAPI
+v3.0 definition files. Read more about [Configuring a REST API using
+OpenAPI](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-import-api.html).
+
+The following code creates an HTTP API using an external OpenAPI definition JSON file -
+
+```ts
+declare const integration: apigateway.Integration;
```

Comment:
Seems like something is missing here. You're not actually using `integration` in this readme and it's not set to any value here.

Reply:
I updated the code to address this comment.